src/krux/display.py

```diff
@@ -304,14 +310,32 @@ def to_lines(self, text, max_lines=None):
             lines.append(text[start:end].rstrip())
             # don't jump space if we're breaking a word
             jump_space = 1 if text[end] == " " else 0
-            start = end + jump_space
+            end += jump_space
+            start = end
             line_count += 1
 
-        # Replace last line with ellipsis if we didn't finish the text
+        # Append/replace ellipsis on last line if we didn't finish the text
         if line_count == max_lines and start < len(text):
-            lines[-1] = lines[-1][: columns - 3] + "..."
-
-        return lines
+            len_last = len(lines[-1])
+            if len_last + 3 > columns:
+                lines[-1] = lines[-1][: columns - 3] + "..."
```

Comment:
We have used two dots ".." instead of three for ellipsis (since the beginning I think). Sometimes the 3 dots appeared and  we later replaced with two only. I think it is a good strategy to have more text and everybody understand that it is ellipsis. What do you think @jdlcdl and @odudex ?

Reply:
I'm ok with two full-stops like "..". 
Unicode has codepoint U2026 as a "horizontal-ellipsis" which might also be a choice/added font glyph "…"